ParkenDD/Colors.swift

```diff
@@ -22,20 +22,34 @@ struct Colors {
 	*/
 	static func colorBasedOnPercentage(_ percentage: Double, emptyLots: Int) -> UIColor {
 
-		let hue = 1 - (percentage * 0.3 + 0.7) // I want to limit this to the colors between 0 and 0.3
+        let lotDensityColor: UIColor = {
+            switch percentage {
+            case 0.6...1:
+                return UIColor.highAvailabilityColor
```

Comment:
Since you're already explicitly stating that `lotDensityColor` is UIColor, you can just `return .highAvailabilityColor`

Reply:
Cool!  Implemented pro tip.